conf/modules.config

```diff
@@ -1113,6 +1126,7 @@ if (!params.skip_multiqc) {
     process {
         withName: 'MULTIQC' {
             ext.args   = params.multiqc_title ? "--title \"$params.multiqc_title\"" : ''
+            ext.prefix = { "$multiqc_report" }
```

Comment:
Where is the variable `multiqc_report` defined in this file?

Reply:
Ahh, this one is a an error- thanks.